sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTests/CloudJobIntegrationTests.cs

```diff
@@ -939,7 +939,60 @@ public async Task Job_GetTaskCounts_ReturnsCorrectCount()
 
                         var counts = await unboundJob.GetTaskCountsAsync().ConfigureAwait(false);
 
-                        Assert.Equal(2, counts.Active);
+                        Assert.Equal(2, counts.TaskCounts.Active);
```

Comment:
Are we testing two different things here? Not sure what the semantic difference is, but if it is two different things, we should probably break it into two different tests.